calc/src/App.css

```diff
@@ -1,3 +1,161 @@
+    }
+    
+    #App {
+        background: rgb(242, 132, 163);
+        background: linear-gradient(140deg, rgba(242, 132, 163, 1) 0%, rgba(208, 72, 153, 1) 100%);
+    }
+    
+    #calc {
+        margin-bottom: 10px;
+        margin-top: 6px;
+        margin-right: 8px;
+        margin-left: 6px;
+    }
+    
+    #screen {
+        height: 100px;
+        background: rgb(78, 89, 103);
+        background: linear-gradient( 90deg, rgba(78, 89, 103, 1) 0%, rgba(70, 81, 95, 1) 48%, rgba(58, 69, 84, 1) 100%, rgba(79, 90, 104, 1) 100%);
+        border-radius: 4px 4px 0px 0px;
+    }
+    
+    #history {
+        margin-bottom: 30px;
+        text-align: right;
+        font-size: 19px;
+        color: #949dac;
+        height: 25px;
+    }
+    
+    .input {
+        margin-bottom: 20px;
+        height: 25px;
+        font-size: 25px;
+        text-align: right;
+        color: #f1f2f3;
+        width: 288px;
+        background: rgb(78, 89, 103);
+        background: linear-gradient( 90deg, rgba(78, 89, 103, 1) 0%, rgba(70, 81, 95, 1) 48%, rgba(58, 69, 84, 1) 100%, rgba(79, 90, 104, 1) 100%);
+        border: transparent;
+    }
+    
+    #buttons {
+        border-bottom-right-radius: 4px;
+        border-bottom-left-radius: 4px;
+        background-color: #384351;
+        width: 288px;
+        height: 310px;
+    }
+    
+    .first {
+        width: 72px;
+        height: 62px;
+    }
+    
+    #zero {
+        width: 72px;
+        height: 62px;
+    }
+    
+    #pm {
+        width: 72px;
+        height: 62px;
+        margin-left: 72px;
+        margin-top: -310px;
+    }
+    
+    .second {
+        width: 72px;
+        height: 62px;
+        margin-left: 72px;
+    }
+    
+    #perc {
+        width: 72px;
+        height: 62px;
+        margin-left: 144px;
+        margin-top: -248px;
+    }
+    
+    .third {
+        width: 72px;
+        height: 62px;
+        margin-left: 144px;
+    }
+    
+    #dzie {
+        width: 72px;
+        height: 62px;
+        margin-left: 216px;
+        margin-top: -310px;
+    }
+    
+    .fourth {
+        width: 72px;
+        height: 62px;
+        margin-left: 216px;
+        justify-content: center;
+        display: flex;
+        align-items: center;
+    }
+    
+    .equal {
+        border-bottom-right-radius: 4px;
+    }
+    
+    .circle {
+        width: 32px;
+        height: 32px;
+        border-radius: 50%;
+        background-color: #485460;
+        text-align: center;
+        font-size: 25px;
+        color: #b0b8c5;
+        justify-content: center;
+        display: flex;
+        align-items: center;
+    }
+    
+    #b0 {
+        border-bottom-left-radius: 4px;
+    }
+    
+    .beq {
+        border-bottom-left-radius: 4px;
+    }
+    
+    #up {
+        color: #5f6a78;
+    }
+    
+    #dz {
+        font-size: 36px;
+    }
+=======
```

Comment:
Do we need this line?

Reply:
Yes because this sign is to small when i set size with another signs is to small compared to the rest

Reply:
yes cuz sign / is small, and this sign must be bigger

Reply:
I talk about 158 line, it is "======="